client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+import analytics from 'lib/analytics';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		analytics.tracks.recordEvent( 'calypso_triforce_partner_redirect', { partner_name: 'Bluehost with WooCommerce' } );
+
+		window.open( 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce' );
+	} else {
+		analytics.tracks.recordEvent( 'calypso_triforce_partner_redirect', { partner_name: 'Bluehost' } );
+
+		window.open( 'https://www.bluehost.com/wordpress' );
```

Comment:
I really like what @aduth suggested. Is there a way we can just make this an HREF attribute on the buttons/links to allow the normal navigation?